chunky/src/java/se/llbit/chunky/world/Chunk.java

```diff
@@ -168,7 +169,8 @@ public synchronized boolean loadChunk(@NotNull Mutable<ChunkData> chunkData, int
 
     surfaceTimestamp = dataTimestamp;
     version = chunkVersion(data);
-    chunkData.set(this.world.createChunkData(chunkData.get(), data.get(DATAVERSION).intValue()));
+    IntIntImmutablePair chunkBounds = inclusiveChunkBounds(data);
+    chunkData.set(this.world.createChunkData(chunkData.get(), chunkBounds.leftInt(), chunkBounds.rightInt()));
```

Comment:
Could we use `yMin` and `yMax` instead of parsing the chunk bounds on every chunk? Although we would probably need to test to see which one is faster.